estate_account/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, Command
+
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def action_sold(self):
+        super().action_sold()
+        for prop in self:
```

Comment:
If you notice that a code may generate issues if self does not contain only 1 record, you can use         `self.ensure_one()` to return an exception if there are multiple records. That way, you make sure that the db will not be corrupted if the method is used on several record. I wrote the comment here but you can do it on pretty much all your python actions of your PR.

```suggestion
    def action_sold(self):
        self.ensure_one()
        super().action_sold()
        for prop in self:
```